libraries/utils.py

```diff
@@ -210,3 +210,62 @@ def legacy_path_transform(content_path):
     if content_path and content_path.startswith(LEGACY_LATEST_RELEASE_URL_PATH_STR):
         content_path = re.sub(r"([a-zA-Z0-9\.]+)/(\S+)", r"latest/\2", content_path)
     return content_path
+
+
+def parse_boostdep_artifact(content: str):
```

Comment:
Can you add a docstring?

Reply:
I think I addressed all of the concerns, let me know if there is anything left.